server/src/config/index.ts

```diff
@@ -9,16 +9,18 @@ interface AppConfig {
   appName: string;
   isProduction: boolean;
   mongoConnection: MongoConnection;
+  draftExpirationTime: number
 }
 
 const appConfig: AppConfig = {
   appName: process.env.appName || 'testing-webapp-api',
   isProduction: process.env.envName ? process.env.envName === 'prod' : false,
   mongoConnection: new MongoConnection(),
+  draftExpirationTime: 24 * 60 * 60 * 1000,
```

Comment:
You should provide information about the draftExpirationTime unit (seconds, minutes, etc.)

Reply:
okay, I will add info like: 
```Javascript
 // Time is specified in milliseconds.
  draftExpirationTime: 7 * 24 * 60 * 60 * 1000, // 7 days```